qiskit_ibm_runtime/estimator.py

```diff
@@ -138,71 +111,36 @@ def __init__(
 
             options: Primitive options, see :class:`Options` for detailed description.
                 The ``backend`` keyword is still supported but is deprecated.
-
-            skip_transpilation: (DEPRECATED) Transpilation is skipped if set to True. False by default.
-                Ignored ``skip_transpilation`` is also specified in ``options``.
         """
         # `self._options` in this class is a Dict.
         # The base class, however, uses a `_run_options` which is an instance of
         # qiskit.providers.Options. We largely ignore this _run_options because we use
         # a nested dictionary to categorize options.
-        super().__init__(
-            circuits=circuits,
-            observables=observables,
-            parameters=parameters,
-        )
-
-        if skip_transpilation:
-            deprecate_arguments(
-                "skip_transpilation",
-                "0.7",
-                "Instead, use the skip_transpilation keyword argument in transpilation_settings.",
-            )
-        if service:
-            deprecate_arguments(
-                "service", "0.7", "Please use the session parameter instead."
-            )
+        super().__init__()
 
         backend = None
         self._session: Session = None
 
         if options is None:
             self._options = asdict(Options())
         elif isinstance(options, Options):
-            skip_transpilation = (
-                options.transpilation.skip_transpilation  # type: ignore[union-attr]
-            )
             self._options = asdict(copy.deepcopy(options))
         else:
             options_copy = copy.deepcopy(options)
-            backend = options_copy.pop("backend", None)
-            if backend is not None:
-                issue_deprecation_msg(
-                    msg="The 'backend' key in 'options' has been deprecated",
-                    version="0.7",
-                    remedy="Please pass the backend when opening a session.",
-                )
             default_options = asdict(Options())
             self._options = Options._merge_options(default_options, options_copy)
             skip_transpilation = self._options.get("transpilation", {}).get(
```

Comment:
Do we need this processing? (From line 132 to 137)

Reply:
You're right, I don't believe this is necessary